misc/python/materialize/cli/ci_coverage_pr_report.py

```diff
@@ -85,10 +93,20 @@ def find_modified_lines() -> Coverage:
                 start = int(parts)
                 length = 1
             for line_nr in range(start, start + length):
-                coverage[file][line_nr] = None
+                coverage[file_path][line_nr] = None
     return coverage
 
 
+def ignore_file_in_coverage_report(file_path: str) -> bool:
+    if file_path.endswith(".rs"):
```

Comment:
`if not` ?

Reply:
Thank you! 🙈
This is what happens when you change the condition in the last second before the commit (changing `include_file...` to `ignore_file...`)

Reply:
✅ Fixed.